external/js/cothority/src/byzcoin/proof.ts

```diff
@@ -136,11 +130,11 @@ export default class Proof extends Message<Proof> {
      * Verify that the proof contains a correct chain from the given genesis. Note that
      * this function doesn't verify the first roster of the chain.
      *
-     * @param genesisID The skipchain ID
+     * @param latestID ID of the latest known and trusted block the proof is based upon
      * @returns an error if something is wrong, null otherwise
      * @deprecated use verifyFrom for a complete verification
      */
-    verify(id: InstanceID): Error {
+    verify(latestID: InstanceID): Error {
```

Comment:
While at it, you can return `Error | undefined` (and change the `null` to `undefined` down there).

Reply:
I don't like too much changing signatures - hope it doesn't break anything.